README.md

```diff
@@ -200,6 +200,7 @@ A sample configuration file with all options is available [here](https://github.
     * `"check-parameters"` disallows unused function and constructor parameters.
         * NOTE: this option is experimental and does not work with classes that use abstract method declarations, among other things. Use at your own risk.
     * `"react"` relaxes the rule for a namespace import named `React` (from either the module `"react"` or `"react/addons"`). Any JSX expression in the file will be treated as a usage of `React` (because it expands to `React.createElement`).
+    * `{"ignore-pattern": "pattern"}` where pattern is a case sensitive regexp. Variable names that matches the pattern will be ignored.
```

Comment:
Just a couple of grammar tweaks would be good here:

* "case sensitive" => "case-sensitive"
* "matches" => "match"